db/schema.rb

```diff
@@ -50,6 +50,10 @@
     t.datetime "created_at",                             :null => false
     t.datetime "updated_at",                             :null => false
     t.string   "role"
+    t.string   "name"
+    t.string   "city"
+    t.string   "country"
```

Comment:
These two don't have much utility either, but I guess they're kind of cute. If we're keeping this we can find a nice jQuery plugin for autocompletion.